DS/UniqueCharactersMain.java

```diff
@@ -0,0 +1,22 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+//Class to call cache function and check unique characters in a string
+public class UniqueCharactersMain {
+	public static void main(String args[]) throws IOException {
+		Scanner scan = new Scanner(System.in);
+		int choice = 0;
+		while (choice != 1) {
+			// Input string from user
+			System.out.println("enter string");
+			String input = null;
```

Comment:
ACTION: Put this in single line String input = scan.nextLine(). Use proper variable name instead of "input"